lib/presentation/breed_layout.dart

```diff
@@ -18,12 +17,6 @@ class BreedLayout extends StatefulWidget {
 class _BreedLayoutState extends State<BreedLayout> {
   final repo = RepositoryImpl(NetworkServiceImpl());
```

Comment:
What is it doing here? Try to remove the line and check if the application will work